challenge1.exs

```diff
@@ -1,3 +1,65 @@
 defmodule Challenge1 do
+  def divisible3(a, b) do
```

Comment:
If this is the main function that tackles the process maybe `get_divisible_by_three_sequence` (or something like that) would be a better name